### PR TITLE
Fix QC remote per detector

### PR DESCRIPTION
**Pull request by miltalex**

### CODE REVIEW

File: workflows/readout-dataflow.yaml [hunk 1]
```diff
@@ -26,6 +26,14 @@ defaults:
     description: "Enable/disable EPN"
     widget: checkBox
     panel: BasicConfiguration
+  qc_remote_workflow: !public # TODO: if false, DCS panel should be disabled
```
Comment (vascobarroso) [inline workflows/readout-dataflow.yaml:29]: @miltalex I guess the TODO should be adapted. What we would like is that if this var is disabled, then all per-detector remote QC workflows vars are disabled.

**Review by vascobarroso — APPROVED** [review]

File: workflows/readout-dataflow.yaml [hunk 2]
```diff
@@ -924,6 +932,12 @@ roles:
           - name: dpl
             enabled: "{{ qc_remote_workflow != 'none' }}"
             include: "{{ qc_remote_workflow }}"
+          - name: echo task
```
Comment (knopers8) [inline workflows/readout-dataflow.yaml:935]: Could you please add a comment explaining what is it for? Without the context that we are aware of (until we forget), this looks very unclear.

Comment (miltalex) [reply workflows/readout-dataflow.yaml:935]: Of course, I will create a second PR

Comment (miltalex) [reply workflows/readout-dataflow.yaml:935]: https://github.com/AliceO2Group/ControlWorkflows/pull/115

**Pull request merged**